rabbit_mq/variables.tf

```diff
@@ -0,0 +1,109 @@
+variable "region" {
+  description = "AWS Default Region"
+  default     = "us-east-1"
+}
+
+variable "profile" {
+  type        = string
+  default     = "default"
+  description = "Credentials profile for AWS provider"
+}
+
+variable "user_tags" {
+  description = "Mandatory tags for the RabbitMQ resources"
+  type        = map(string)
+}
+
+variable "octopus_tags" {
+  description = "Octopus Tags"
+  type        = map(string)
+}
+
+variable "broker_name" {
+  type        = string
+  description = "Name of the Amazon MQ Broker"
+}
+
+variable "engine_type" {
+  default     = "RabbitMQ"
+  description = "This terraform module will currently support only RabbitMQ"
+}
+
+variable "engine_version" {
+  type        = string
+  default     = "3.8.11"
+  description = "This is the engine version for RabbitMQ"
+  validation {
+    condition     = contains(["3.8.6", "3.8.11"], lower(var.engine_version))
+    error_message = "Unsupported engine version for AWS MQ where engine type is RabbitMQ."
+  }
+}
+
+variable "broker_instance_type" {
+  type        = string
+  default     = "mq.m5.large"
+  description = "This is the broker instance type for RabbitMQ"
+  validation {
```

Comment:
Same as above. Doesn't AWS handle the validation?

Reply:
removed